core/services/s4/postgres_orm.go

```diff
@@ -0,0 +1,95 @@
+package s4
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/sqlx"
+)
+
+type postgres struct {
+	q         pg.Q
+	tableName string
+}
+
+var _ ORM = (*postgres)(nil)
+
+const fields = "address, slot_id, version, expiration, confirmed, payload, signature, updated_at"
+
+func NewPostgresORM(db *sqlx.DB, lggr logger.Logger, cfg pg.QConfig, tableName string) ORM {
```

Comment:
Isn't the convention to put logger last?

Reply:
I cannot remember such a convention.. I just followed `pg.NewQ` params order.